.vsts-ci.yml

```diff
@@ -116,7 +117,7 @@ extends:
               condition: and(eq(variables['system.pullrequest.isfork'], false), eq(variables['_BuildConfig'], 'Release'))
               inputs:
                 PathtoPublish: '$(Build.ArtifactStagingDirectory)'
-                artifactName: 'drop-windows'
```

Comment:
This was being ignored due to incorrect casing, and all artifacts (mac-arm64, mac-x64 and windows) were being grouped together

Reply:
Good find! Input name casing is significant in AzDO pipelines.